pubspec.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
   flutter:
     sdk: flutter
   dio: ^3.0.10
-  meta: ^1.2.3
+  meta:
```

Comment:
Following the guidelines written in https://dart.dev/tools/pub/versioning, is it wise not to include specific versions of the packages _http-mock-adapter_ uses?

As far as I know, having a concrete version of a library is safer for the developers because they will be sure that the package they use (_http-mock-adapter_) will have defined versions of the packages it uses, which avoids the problem with breaking changes if the dependent packages do get updated with breaking changes.

What do you think? am I misinformed?

Reply:
კი,კი,ვიცი ეგ და გეთანხმები flutter test-ის ვერსიასთან მოდიოდა კონფლიქტში და მაგიტო წავშალე.რა თქმა უნდა,ერთ ვერსიაზე უნდა დავსვათ.